# Making sense of genomic data: COVID-19 web-based bioinformatics

![](images/OC3_cover.jpeg)

## About the course         

Over the past decade, sequencing technologies have become more accessible and sequencing data has increased exponentially, especially due to the COVID-19 pandemic. However, bioinformatics capacity is still limited. This course will provide concepts and tools for web-based SARS-CoV-2 analyses, aiming to mitigate the bottleneck on sequencing analyses.


**Learning outcomes**         

* Describe the SARS-CoV-2 viral genomic structure         
* Evaluate different sequence analysis outputs           
* Apply a basic analytical web-based pipeline            
* Examine how variant calling contributes to epidemiological inferences           

**Target audience**            

This course is designed for diagnostic and healthcare professionals, researchers, and anyone involved in the testing and analysis of disease samples. It will also be advantageous to researchers specialising in web-based bioinformatics, diagnostics, diseases, or pandemics.

## Content

**[Week 1 - Introduction to viral genomics](https://wcscourses.github.io/COG-Train_Resources/making_sense_week1.html#INTRODUCTION_TO_VIRAL_GENOMICS)**

* [Glossary](https://wcscourses.github.io/COG-Train_Resources/making_sense_week1.html#Glossary)      
* [Frequently asked questions](https://wcscourses.github.io/COG-Train_Resources/making_sense_week1.html#Frequently_asked_questions)         
* [SARS-CoV-2 replication cycle](https://wcscourses.github.io/COG-Train_Resources/making_sense_week1.html#SARS-CoV-2_replication_cycle)       
* [SARS-CoV-2 genomic landscape](https://wcscourses.github.io/COG-Train_Resources/making_sense_week1.html#SARS-CoV-2_genomic_landscape)           
* [Overview of genomic, sub-genomic and anti-genomic sequences](https://wcscourses.github.io/COG-Train_Resources/making_sense_week1.html#Overview_of_genomic,_sub-genomic_and_anti-genomic_sequences)        
* [Introduction to sequencing methods](https://wcscourses.github.io/COG-Train_Resources/making_sense_week1.html#Introduction_to_sequencing_methods)                 
* [How is SARS-CoV-2 sequencing done?](https://wcscourses.github.io/COG-Train_Resources/making_sense_week1.html#How_is_SARS-CoV-2_sequencing_done)         
* [Amplicon-based sequencing](https://wcscourses.github.io/COG-Train_Resources/making_sense_week1.html#Amplicon-based_sequencing)              
* [Overview of different data formats](https://wcscourses.github.io/COG-Train_Resources/making_sense_week1.html#Overview_of_different_data_formats)                  
* [What is bioinformatics?](https://wcscourses.github.io/COG-Train_Resources/making_sense_week1.html#What_is_bioinformatics)                 
* [Overview of web-based bioinformatics analyses using Galaxy](https://wcscourses.github.io/COG-Train_Resources/making_sense_week1.html#Overview_of_web-based_bioinformatics_analyses_using_Galaxy)                   
* [Importing data onto Galaxy](https://wcscourses.github.io/COG-Train_Resources/making_sense_week1.html#Importing_data_onto_Galaxy)                     
* [Processing data on Galaxy](https://wcscourses.github.io/COG-Train_Resources/making_sense_week1.html#Processing_data_on_Galaxy)                  
* [Building a workflow on Galaxy](https://wcscourses.github.io/COG-Train_Resources/making_sense_week1.html#Building_a_workflow_on_Galaxy)                  
* [Using existing workflows on Galaxy](https://wcscourses.github.io/COG-Train_Resources/making_sense_week1.html#Using_existing_workflows_on_Galaxy)                    

**[Week 2 - Exploring the pipelines available](https://wcscourses.github.io/COG-Train_Resources/making_sense_week2.html#EXPLORING_THE_PIPELINES_AVAILABLE)**      

* [Garbage in/Garbage out: importance of generating good data and cleaning up](https://wcscourses.github.io/COG-Train_Resources/making_sense_week2.html#Garbage_inGarbage_out:_importance_of_generating_good_data_and_cleaning_up)           
* [Data cleaning and quality contro](https://wcscourses.github.io/COG-Train_Resources/making_sense_week2.html#Data_cleaning_and_quality_control)       
* [FastQC and MultiQC tools](https://wcscourses.github.io/COG-Train_Resources/making_sense_week2.html#FastQC_and_MultiQC_tools)            
* [Assess quality with FastQC](https://wcscourses.github.io/COG-Train_Resources/making_sense_week2.html#Assess_quality_with_FastQC)            
* [Assembly of SARS-CoV-2 genome and sequence alignment](https://wcscourses.github.io/COG-Train_Resources/making_sense_week2.html#Assembly_of_SARS-CoV-2_genome_and_sequence_alignment)             
* [Assembly tutorial](https://wcscourses.github.io/COG-Train_Resources/making_sense_week2.html#Assembly_tutorial)            
* [Variant calling and annotation](https://wcscourses.github.io/COG-Train_Resources/making_sense_week2.html#Variant_calling_and_annotation)              
* [Variant calling tutoria](https://wcscourses.github.io/COG-Train_Resources/making_sense_week2.html#Variant_calling_tutorial)              
* [Using and interpreting Nextclade](https://wcscourses.github.io/COG-Train_Resources/making_sense_week2.html#Using_and_interpreting_Nextclade)         
* [Signatures of co-infection, recombination and contamination](https://wcscourses.github.io/COG-Train_Resources/making_sense_week2.html#Signatures_of_co-infection,_recombination_and_contamination)          
* [What are clades, lineages and variants?](https://wcscourses.github.io/COG-Train_Resources/making_sense_week2.html#What_are_clades,_lineages_and_variants)             
* [Prediction of minor variants and drug resistance](https://wcscourses.github.io/COG-Train_Resources/making_sense_week2.html#Prediction_of_minor_variants_and_drug_resistance)             
* [Identification of recombinants](https://wcscourses.github.io/COG-Train_Resources/making_sense_week2.html#Identification_of_recombinants)            
* [Introduction to phylogenetics: tools and models](https://wcscourses.github.io/COG-Train_Resources/making_sense_week2.html#Introduction_to_phylogenetics:_tools_and_models)               
* [Transmission detection: travel-associated, hospital and community transmission](https://wcscourses.github.io/COG-Train_Resources/making_sense_week2.html#Transmission_detection:_travel-associated,_hospital_and_community_transmission)              
* [Tutorial on phylogenetics](https://wcscourses.github.io/COG-Train_Resources/making_sense_week2.html#Tutorial_on_phylogenetics)            
* [Quality control in phylogenetics](https://wcscourses.github.io/COG-Train_Resources/making_sense_week2.html#Quality_control_in_phylogenetics)               


**[Week 3 - Introduction to genomic epidemiology of COVID-19](https://wcscourses.github.io/COG-Train_Resources/making_sense_week3.html#INTRODUCTION_TO_GENOMIC_EPIDEMIOLOGY_OF_COVID-19)**
* [The importance of sample and data consent](https://wcscourses.github.io/COG-Train_Resources/making_sense_week3.html#The_importance_of_sample_and_data_consent)        
* [Ethics in data sharing](https://wcscourses.github.io/COG-Train_Resources/making_sense_week3.html#Ethics_in_data_sharing)        
* [Metadata: standardisation and collection tools](https://wcscourses.github.io/COG-Train_Resources/making_sense_week3.html#Metadata:_standardisation_and_collection_tools)              
* [Ownership of analyses tools and pipelines](https://wcscourses.github.io/COG-Train_Resources/making_sense_week3.html#Ownership_of_analyses_tools_and_pipelines)              
* [Best practices for data sharing](https://wcscourses.github.io/COG-Train_Resources/making_sense_week3.html#Best_practices_for_data_sharing)             
* [Data quality metrics](https://wcscourses.github.io/COG-Train_Resources/making_sense_week3.html#Data_quality_metrics)         
* [Pros and cons of public databases for data sharing](https://wcscourses.github.io/COG-Train_Resources/making_sense_week3.html#Pros_and_cons_of_public_databases_for_data_sharing)          
* [When and why to obtain sequencing data](https://wcscourses.github.io/COG-Train_Resources/making_sense_week3.html#When_and_why_to_obtain_sequencing_data)          
* [How and where to obtain the public data](https://wcscourses.github.io/COG-Train_Resources/making_sense_week3.html#How_and_where_to_obtain_the_public_data)            
* [Challenges and limitations of data sharing](https://wcscourses.github.io/COG-Train_Resources/making_sense_week3.html#Challenges_and_limitations_of_data_sharing)            
* [Best practices for presenting your data](https://wcscourses.github.io/COG-Train_Resources/making_sense_week3.html#Best_practices_for_presenting_your_data)          
* [Public health reports and sharing](https://wcscourses.github.io/COG-Train_Resources/making_sense_week3.html#Public_health_reports_and_sharing)         
* [Resources](https://wcscourses.github.io/COG-Train_Resources/making_sense_week3.html#Resources)  

## Collaborators

**Educators**     

Carolina Castañeda Garcia, International Laboratory for Human Genome Research, Universidad Nacional Autónoma de México, Mexico                 
Claudine Nkera-Gutabara, Sydney Brenner Institute for Molecular Bioscience, Division of Human Genetics, University of Witwatersrand, South Africa             
Tanya Golubchik, Sydney Institute for Infectious Diseases, University of Sydney, Australia           
Tracey Calvert-Joshua, South African National Bioinformatics Institute, South Africa          
Varun Shamanna, Central Research Laboratory, Kempegowda Institute of Medical Sciences, India             
             
**Contributors**             

Ana Villasenor Altamirano, Brigham and Women’s Hospital, Harvard Medical School, United States                   
Brian Kariithi, University of the Witwatersrand, South Africa                
David Twesigomwe, Sydney Brenner Institute for Molecular Bioscience, South Africa        
Eunice Niyobuntu, University of the Witwatersrand, South Africa        
Merlin Davies, University of Exeter, United Kingdom                   
Muthumeenakshi Bhaskaran, Central Research Laboratory, KIMS, India           
Nischita Subramanya, Central Research Laboratory, KIMS, India                
Rebecca Rockett, University of Sydney, Australia                  
Shreedhanya Marathe, Central Research Laboratory, KIMS, India           
 
**Reviewers**

Arporn (Koi) Wangwiwatsin, Cholangiocarcinoma Research Institute and Khon Kaen University, Thailand            
Andrés Culasso, University of Buenos Aires and Consejo Nacional de Investigaciones Científicas y Tecnológicas, Argentina          
Daniel Gyamfi Amoako, National Institute for Communicable Diseases, South Africa          

**Education developers**           

Liã Bárbara Arruda, Wellcome Connecting Science, United Kingdom       
Jorge Batista da Rocha, Wellcome Connecting Science, United Kingdom          
 
**COG-UK contributors**
 
Sharon Peacock, University of Cambridge, United Kingdom          
Alistair Darby, University of Liverpool, United Kingdom              
Darren Smith, Northumbria University, United Kingdom             
Ewan Harrison, Wellcome Sanger Institute, United Kingdom         
Leigh Jackson, University of Exeter, United Kingdom          
Anna Markov, University of Cambridge, United Kingdom           
Ellena Brooks, University of Cambridge, United Kingdom          
Kim Smith, University of Cambridge, United Kingdom         
Laia Delgado Callico, University of Cambridge, United Kingdom            
Peter McEwan, University of Cambridge, United Kingdom               

**Wellcome Connecting Science contributors**

Alice Matimba, Wellcome Connecting Science, United Kingdom            
Dusanka Nikolic, Wellcome Connecting Science, United Kingdom            
Mel Sharpe, Wellcome Connecting Science, United Kingdom    
Rachel Berkson, Wellcome Connecting Science, United Kingdom            
Treasa Creavin, Wellcome Connecting Science, United Kingdom              
              
## Original version

**Original platform:** FutureLearn       
**Original course page:** [Making sense of genomic data: COVID-19 web-based bioinformatics](https://www.futurelearn.com/courses/making-sense-of-genomic-data-covid-19-web-based-bioinformatics/1)                            
**Launch of the original version:** 25 July 2022                
**Number of joiners:** 2,488         
**Number of countries reached:** 136         
**Review score:** 4.6/5 (26 reviews)         

******
Any reuse of the course materials is encouraged with due acknowledgement.

## License
<a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons Licence" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.

<!-- ## How to cite 

TBP --> 









