# A PRACTICAL GUIDE FOR SARS-COV-2 WHOLE-GENOME SEQUENCING

![](images/OC4_cover.jpeg)

## About the course

SARS-CoV-2 genomics surveillance has been proven pivotal for the management and control of the COVID-19 pandemic. However, genomics capacity is still limited, particularly in low-and-middle-income countries (LMICs). This course will provide tools and resources for capacity building aiming to strengthen the genomics surveillance approaches


**Learning outcomes**

* Compare sequencing methodologies
* Critically evaluate different approaches to determine SARS-CoV-2 genomes and variants
* Identify ways in which to scale up SARS-CoV-2 sequencing
* Describe SARS-CoV-2 analysis pipelines         

**Target audience**

This course is designed for diagnostic and healthcare professionals, researchers, and anyone involved in the testing and analysis of COVID-19 samples.

## Content

**[Week 1 Fundamentals of viral sequencing](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week1.html#FUNDAMENTALS_OF_VIRAL_SEQUENCING)    

* [Glossary](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week1.html#Glossary)     
* [Introduction to viral genomes](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week1.html#Introduction_to_viral_genomes)         
* [Learning more about SARS-CoV-2 genome](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week1.html#Learning_more_about_SARS-CoV-2_genome)       
* [Why viral genome sequencing is important](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week1.html#Why_viral_genome_sequencing_is_important)         
* [How to sequence using Illumina technology](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week1.html#How_to_sequence_using_Illumina_technology)      
* [Why was Nanopore sequencing used in the SARS-CoV-2 pandemic?](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week1.html#Why_was_Nanopore_sequencing_used_in_the_SARS-CoV-2_pandemic)        
* [Using metagenomics to investigate infections](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week1.html#Using_metagenomics_to_investigate_infections)         
* [What is viral whole-genome sequencing](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week1.html#What_is_viral_whole-genome_sequencing)       
* [Using sequencing to investigate SARS-CoV-2](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week1.html#Using_sequencing_to_investigate_SARS-CoV-2)       
* [SARS-CoV-2 sequencing workflow](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week1.html#SARS-CoV-2_sequencing_workflow)         
* [Why you should collect metadata](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week1.html#Why_you_should_collect_metadata)        
* [How to handle primary samples](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week1.html#How_to_handle_primary_samples)        
* [Storing high-quality samples](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week1.html#Storing_high-quality_samples)           
* [What happens to samples after sequencing](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week1.html#What_happens_to_samples_after_sequencing)         
* [Assessing the quality of nucleic acid](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week1.html#Assessing_the_quality_of_nucleic_acid)        

**[Week 2 Library preparation and sequencing](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week2.html#LIBRARY_PREPARATION_AND_SEQUENCING)**        

* [How to isolate good quality nucleic acids](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week2.html#How_to_isolate_good_quality_nucleic_acids)       
* [What a targeted sequencing protocol](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week2.html#What_a_targeted_sequencing_protocol)          
* [How SARS-CoV-2 primer sets were designed](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week2.html#How_SARS-CoV-2_primer_sets_were_designed)         
* [Producing SARS-CoV-2 amplicons](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week2.html#Producing_SARS-CoV-2_amplicons)         
* [Checking the quality of amplicons](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week2.html#Checking_the_quality_of_amplicons)           
* [What is a sequencing library and how is it prepared?](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week2.html#What_is_a_sequencing_library_and_how_is_it_prepared)            
* [Comparing library preparation methods](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week2.html#Comparing_library_preparation_methods)          
* [Illumina: pooling samples and loading cartridges](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week2.html#Illumina:_pooling_samples_and_loading_cartridges)         
* [Nanopore: pooling samples and loading flow cells](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week2.html#Nanopore:_pooling_samples_and_loading_flow_cells)         
* [How SARS-CoV-2 has been sequenced around the world](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week2.html#How_SARS-CoV-2_has_been_sequenced_around_the_world)          
* [Comparing manual and automated methodologies](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week2.html#Comparing_manual_and_automated_methodologies)          
* [How to avoid contamination](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week2.html#How_to_avoid_contamination)      
* [Coordinating human resources in a pandemic](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week2.html#Coordinating_human_resources_in_a_pandemic)        
* [How to plan ordering and tracking of reagents](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week2.html#How_to_plan_ordering_and_tracking_of_reagents)       
* [Legacies of the COVID-19 pandemic](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week2.html#Legacies_of_the_COVID-19_pandemic)           

**[Week 3 Sequencing analysis and interpretation](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week3.html#SEQUENCING_ANALYSIS_AND_INTERPRETATION)**

* [What are sequencing metrics[(https://wcscourses.github.io/COG-Train_Resources/practical_guide_week3.html#What_are_sequencing_metrics)       
* [Accessing and using sequencing data](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week3.html#Accessing_and_using_sequencing_data)       
* [How to generate a consensus sequence](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week3.html#How_to_generate_a_consensus_sequence)        
* [User-friendly tools for quality control](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week3.html#User-friendly_tools_for_quality_control)        
* [Why high-quality data is important](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week3.html#Why_high-quality_data_is_important)        
* [What are clades, lineages and variants?](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week3.html#What_are_clades,_lineages_and_variants)        
* [SARS-CoV-2 Variants](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week3.html#SARS-CoV-2_Variants)       
* [Interpreting Nextclade quality metrics](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week3.html#Interpreting_Nextclade_quality_metrics)        
* [Lineages classification using Pangolin](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week3.html#Lineages_classification_using_Pangolin)         
* [Introduction to phylogenetics: tools and models](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week3.html#Introduction_to_phylogenetics:_tools_and_models)          
* [How to detect an outbreak](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week3.html#How_to_detect_an_outbreak)        
* [Generating public health reports](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week3.html#Generating_public_health_reports)       
* [How to prevent sequencing and sampling bias](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week3.html#How_to_prevent_sequencing_and_sampling_bias)      
* [Using sequencing data to identify outbreaks in a hospital](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week3.html#Using_sequencing_data_to_identify_outbreaks_in_a_hospital)       
* [The importance of sample and data consent](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week3.html#The_importance_of_sample_and_data_consent)        
* [Establishing collaborations in a pandemic](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week3.html#Establishing_collaborations_in_a_pandemic)      
* [Challenges and how to overcome them](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week3.html#Challenges_and_how_to_overcome_them)       
* [Resources](https://wcscourses.github.io/COG-Train_Resources/practical_guide_week3.html#Resources)       

## Collaborators

**Educators**     

Angie Beckett, University of Portsmouth, United Kingdom     
Charlotte Williams, University College London Genomics (UCLG), United Kingdom     
Geetha Nagaraj, Central Research Laboratory, Kempegowda Institute of Medical Sciences (CRL, KIMS), India     
Pablo Tsukayama, Universidad Peruana Cayetano Heredia, Peru     
Rachel Williams, University College London Genomics (UCLG), United Kingdom                
             
**Contributors**             

Christian Happi, African Centre of Excellence for Genomics of Infectious Diseases (ACEGID), Redeemer's University, Nigeria     
Geetika Patel, Parul Sevashram Hospital, Parul Institute of Medical Sciences & Research, India     
Hannah Dent, University of Portsmouth, United Kingdom    
Helena Tutill, University College London, United Kingdom    
Jo Herbert, University of Portsmouth, United Kingdom    
Judith Breuer, University College London, United Kingdom    
Julianne Brown, Great Ormond Street Hospital for Children NHS Foundation Trust, United Kingdom    
Kate Cook, Ineos Oxford Institute, University of Oxford, United Kingdom     
Krunal Shah, Parul Institute of Medical Sciences and Research, Parul University, India     
Leysa Forrest, University College London, United Kingdom      
Luz Marina Martin Bernal, University College London, United Kingdom        
Murali Dharan Bashyam, Centre for DNA Fingerprinting and Diagnostics (CDFD), India     
Muthumeenakshi Bhaskaran, Central Research Laboratory, Kempegowda Institute of Medical Sciences (KIMS), India        
Naomi Park, Wellcome Sanger Institute, United Kingdom        
Oscar Enrique Torres Montaguth, University College London, United Kingdom         
Paola Niola, University College London, United Kingdom          
Paola Resende, Oswaldo Cruz Foundation (FIOCRUZ), Brazil         
Patricia Dyal, University College London, United Kingdom         
Ramalingam Bethunaickan, Indian Council of Medical Research (ICMR), National Institute for Research in Tuberculosis, India        
Richard Orton, Medical Research Council, University of Glasgow, Centre for Virus Research, United Kingdom        
Rijo Joseph John, Central Research Laboratory, Kempegowda Institute of Medical Sciences (KIMS), India         
Sam Robson, University of Portsmouth, United Kingdom          
Sandra Cantilena, Great Ormond Street Hospital for Children NHS Foundation Trust, United Kingdom         
Senjuti Saha, Child Health Research Foundation, Bangladesh          
Tony Brooks, University College London, United Kingdom          
Varun Shamanna, Central Research Laboratory, Kempegowda Institute of Medical Sciences (KIMS), India           
 
**Reviewers**

Ana Carolina Soares de Oliveira, University of São Paulo, Brazil        
Ariel Amadio, Universidad Nacional de Rafaela and IDICAL, INTA-CONICET, Argentina       
Matthew Cotten, MRC/UVRI & London School of Hygiene and Tropical Medicine, Uganda        

**Education developer**

Liã Bárbara Arruda, Wellcome Connecting Science, United Kingdom        
 
**COG-UK contributors**
 
Sharon Peacock, University of Cambridge, United Kingdom          
Alistair Darby, University of Liverpool, United Kingdom         
Darren Smith, Northumbria University, United Kingdom         
Catherine Ludden, Genomics England, United Kingdom         
Leigh Jackson, University of Exeter, United Kingdom          
Anna Markov, University of Cambridge, United Kingdom           
Ellena Brooks, University of Cambridge, United Kingdom        
Kim Smith, University of Cambridge, United Kingdom         
Laia Delgado Callico, University of Cambridge, United Kingdom          
Peter McEwan, University of Cambridge, United Kingdom            

**Wellcome Connecting Science contributors**

Alice Matimba, Wellcome Connecting Science, United Kingdom      
Dusanka Nikolic, Wellcome Connecting Science, United Kingdom         
Jorge Batista da Rocha, Wellcome Connecting Science, United Kingdom          
Mel Sharpe, Wellcome Connecting Science, United Kingdom       
Rachel Berkson, Wellcome Connecting Science, United Kingdom        
Treasa Creavin, Wellcome Connecting Science, United Kingdom          
              
## Original version

**Original platform:** FutureLearn       
**Original course page:** [https://www.futurelearn.com/courses/a-practical-guide-for-sars-cov-2-whole-genome-sequencing/1](https://www.futurelearn.com/courses/a-practical-guide-for-sars-cov-2-whole-genome-sequencing/1)                           

Data collected on 31 May 2023         
**Number of joiners:** 2,011            
**Number of countries reached:** 128            
**Review score:** 4.6/5 (31 reviews)               

******
Any reuse of the course materials is encouraged with due acknowledgement.

## License
<a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons Licence" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.

## How to cite 

COG-Train. (2022). A practical guide for SARS-CoV-2 whole-genome sequencing. Zenodo. https://doi.org/10.5281/zenodo.8200187 


